### PR TITLE
9.24.7 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.24.md [hunk 1]
```diff
@@ -9,6 +9,31 @@ weight: 76
...
+* We fixed an issue in the $metadata of published OData services using OData version 3 where a self-referencing association could result in two roles with the same name (Ticket 193467).
+* We fixed an issue where a text area would ignore a value autofilled in Chrome. (Ticket 176195)
+* We fixed an issue where the grid control does not highlight the selected row.
+* We fixed an issue where setting a new authtoken would not apply until a restart of the native app. (Ticket 195005)
+
+### Deprecations
+
+* We will drop support for PostgreSQL 11 after November 1, 2023, as it is no longer supported by the vendor.
+* We fixed an issue where it was not possible to use the [constrained by](https://docs.mendix.com/refguide/reference-selector/#selectable-objects) functionality, with [Reference selector](https://docs.mendix.com/refguide/reference-selector/) and [Input reference set selector](https://docs.mendix.com/refguide/input-reference-set-selector/) widgets when the configured attribute path did not start from the direct enclosing data container.
```
Comment (ancajuravle) [inline content/en/docs/releasenotes/studio-pro/9/9.24.md]: This should go under Fixes, and not Deprecations (I am afraid the release notes auto-merge didn't go that well). Can this be adjusted or should I open a separate PR?

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.24.md]: I'll adjust, thanks @ancajuravle
